src/service/http-server.ts

```diff
@@ -160,6 +160,8 @@ export class HttpServer {
       timezone: req.query.timezone,
       encoding: req.query.encoding,
       deviceScaleFactor: req.query.deviceScaleFactor,
+      fullPageImage: req.query.fullPageImage,
```

Comment:
What's the reason behind adding this line? I don't see this query parameter being filled on the Grafana side.